packages/dx-react-grid/docs/reference/table-group-header.md

```diff
@@ -0,0 +1,64 @@
+# TableGroupHeader Plugin Reference
+
+A plugin that renders the nested header cells.
+
+## User Reference
+
+### Dependencies
+
+- [Table](table.md)
+- [TableHeaderRow](table-header-row.md)
+- [TableEditColumn](table-edit-column.md) [Optional]
+- [TableSelection](table-selection.md) [Optional]
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+cellComponent | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a parent cell.
+rowComponent | ComponentType&lt;[Table.RowProps](table.md#tablerowprops)&gt; | | A component that renders row for parent cells.
+enhancerHeaderCell | ComponentType&lt;[TableGroupHeader.EnhancerHeaderCellProps](#tablegroupheaderenhancerheadercellprops)&gt; | | A enhancer component for a header cell.
```

Comment:
"Enhancer"? What is it? What is it for? What is it able to do?

Reply:
This component renders HeaderCell and passes to HeaderCell `rowSpan` and `style` properties. This looks like middleware